clarity/enhancer/nalr.py

```diff
@@ -135,15 +135,19 @@ def hl_interp(self, hl, cfs):
         hl_interpf = scipy.interpolate.interp1d(cfs, hl)
         return hl_interpf(self.aud)
 
-    def build(self, HL, cfs):
+    def build(self, HL, cfs=None):
```

Comment:
Rather than having this as `None` you could set the default here directly and do away with the `if()` statement.

```suggestion
    def build(self, HL: np.ndarray, cfs: np.ndarray=np.array([250, 500, 1000, 2000, 3000, 4000, 6000, 8000])):
```